<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация</string>
  </property>
  <widget class="QLabel" name="count_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Колличество частей речи:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="noun_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя существительное:</string>
   </property>
  </widget>
  <widget class="QLabel" name="adjf_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя прилагательное:</string>
   </property>
  </widget>
  <widget class="QLabel" name="verb_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Глагол:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prtf_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Причастие:</string>
   </property>
  </widget>
  <widget class="QLabel" name="grnd_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Деепричастие:</string>
   </property>
  </widget>
  <widget class="QLabel" name="advb_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Наречие:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prep_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Предлог:</string>
   </property>
  </widget>
  <widget class="QLabel" name="numr_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Числительное:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prcl_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Частица:</string>
   </property>
  </widget>
  <widget class="QLabel" name="conj_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Союз:</string>
   </property>
  </widget>
  <widget class="QLabel" name="popular_words_label">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>379</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Самые популярные слова (начальная форма):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="list_popular_words">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>361</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="popular_words">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Самые популярные слова частей речи:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="noun_label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя существительное:</string>
   </property>
  </widget>
  <widget class="QLabel" name="adjf_label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя прилагательное:</string>
   </property>
  </widget>
  <widget class="QLabel" name="verb_label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>350</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Глагол:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prtf_label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Причастие:</string>
   </property>
  </widget>
  <widget class="QLabel" name="grnd_label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>630</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Деепричастие:</string>
   </property>
  </widget>
  <widget class="QLabel" name="numr_label_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Числительное:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="adjf_list">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="verb_list">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="prtf_list">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="grnd_list">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>660</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="noun_list">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="prep_list">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>380</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="prcl_list">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>660</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="advb_list">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>240</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="conj_list">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>520</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="numr_list">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>100</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="advb_label_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Наречие:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prep_label_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>350</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Предлог:</string>
   </property>
  </widget>
  <widget class="QLabel" name="conj_label_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>490</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Союз:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prcl_label_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>630</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Частица:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
